pkg/cmd/opts/common.go

```diff
@@ -8,50 +8,45 @@ import (
 	"strings"
 	"time"
 
-	"github.com/jenkins-x/jx/pkg/config"
-
-	"github.com/jenkins-x/jx/pkg/kube/cluster"
-
-	gojenkins "github.com/jenkins-x/golang-jenkins"
-	"github.com/jenkins-x/jx/pkg/cloud/gke"
-	"github.com/jenkins-x/jx/pkg/prow"
-	"github.com/jenkins-x/jx/pkg/versionstream"
-	"github.com/spf13/viper"
-
-	"github.com/jenkins-x/jx/pkg/secreturl"
-	"github.com/spf13/pflag"
-
+	vaultoperatorclient "github.com/banzaicloud/bank-vaults/operator/pkg/client/clientset/versioned"
```

Comment:
this here is just a import ordering change, right? Do you have any custom setting for that? Just wondering?

Reply:
No. In my previous work, I used to sort them as:

- std lib
- line break
- external dependency
- line break
- internal dependency

After that, I run `go import` in the goland IDE, so that it sorts them in alphabetical order.

Reply:
no worries